### PR TITLE
BREAKING CHANGES: Auto SAS Tokens

### PR DESCRIPTION
**Pull request by futurethang**
This PR pulls the token generation logic into the widgets themselves for a more obscured security token and easier usage experience.

### CODE REVIEW

**Comment by futurethang** [comment]
@venkateshwarv I did not as of yet apply this to ActionBuilder, but it may very well apply just as well!

**Review by elenagiere — APPROVED** [review]
Looks good to me!

File: ProfileView/src/index.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export default class CjaasProfileWidget extends LitElement {
     | null = null;
   /**
    * SAS Token for POST operations on Profile endpoint
-   * @attr profile-write-token
+   * @attr profile-token
```
Comment (venkateshwarv) [inline ProfileView/src/index.ts:79]: Why are we getting rid of permission info from attribute name. It would explicitly inform user on what token to use.

**Comment by venkateshwarv** [comment]
> 
> 
> @venkateshwarv I did not as of yet apply this to ActionBuilder, but it may very well apply just as well!

No hurry.

File: ProfileView/src/index.ts (hunk 1 above)
Comment (futurethang) [reply ProfileView/src/index.ts:79]: So what's your opinion, should the widget be so explicit as to have a specific token for read and write? Or rely upon docs to tell users that this widget needs both permissions, and allow that to happen with one token . . . then we have one token to rule them all, and confusion creeps back in when a user wonders if 'profile-write-token' automatically implies that it reads as well.

So long story short, I prefer "provide one token" and docs say "make the token read and write"

**Pull request merged**